### ISSUE TITLE
Add support for Progress OpenEdge Database

### ISSUE THREAD

**Issue by j0hannesr0th** [apache/superset#31594]
### Motivation

Currently there's no support for Progress OpenEdge. It has been asked a few years ago already: https://github.com/apache/superset/issues/2874

### Proposed Change

Example for SQL execution:
 https://community.progress.com/s/article/sample-python-code-to-access-an-openedge-db-through-odbc 

### New dependencies

pyodbc

### Migration Plan and Compatibility

None.

### Rejected Alternatives

-

**Comment by rusackas** [comment]
I don't think we need a SIP for this... please feel free to open a PR adding support at any time.

**Comment by rusackas** [comment]
I'll close this, and we welcome you to add support for OpenEdge. If you need help with this, I can reopen the issue, but would convert it to an "Ideas" discussion.
